local.go

```diff
@@ -297,6 +307,18 @@ func checkConvergence(loc *Location, iterType IterationType, stats *Stats, setti
 	return NotTerminated
 }
 
+func invalidate(loc *Location, f, grad, hess bool) {
```

Comment:
Could you add a quick function comment? It's clear what the function does, so the comment just needs to say why.